content/guides/getstarted/setup-local-development.md

```diff
@@ -38,48 +38,266 @@ deploying Solana programs to the blockchain.
 - how to create a Solana wallet for developing
 - how to airdrop SOL tokens for your wallet
 
-## Windows users only
+If you are very new to development in general, or have just tried EVM based blockchains or just have fiddled with Nodejs and reactJS, your machine is not yet ready to help you code on solana.
```

Comment:
Minor but remember to capitalize React, Node.js, Windows, etc. macOS is just macOS though according to Apple.